docs/pages/config-reference.mdx

```diff
@@ -48,6 +54,17 @@ teleport:
         - 10.1.0.5:3025
         - 10.1.0.6:3025
 
+    # For use on a Teleport Proxy
+    # See the "Teleport Scalability Tweaks" of the admin manual
+    # (https://goteleport.com/teleport/docs/admin-guide/#teleport-scalability-tweaks)
+    #cache:
+    #  # use an in-memory cache to speed up the connection of many teleport nodes
+    #  # back to proxy
+    #  type: in-memory
+    #  enabled: true
+    #  # can be never or a golang time.ParseDuration string
```

Comment:
let's show a real format examples here for people who don't know golang

Reply:
```
    #  # can be "never" or a duration such as 300m (300 minutes),
    #  # 2.5h (2 and a half hours), etc. Default is 20h (twenty hours).
```

I pulled the default from https://github.com/gravitational/teleport/blob/94acec4c84db920fbb45f62b00987f3160ccea47/lib/defaults/defaults.go#L85 which I'm only about 95% sure is correct.